cif/products/src/helpers/CommerceIntegrationHelper.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2019 diconium
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export default class CommerceIntegrationHelper {
```

Comment:
Why do we have a helper if we use this code only inside one class?

Reply:
The filters have many types and the list is likely to grow in future and I did not want to maintain it in the mapper class. So i moved it to this helper class which would have an enum for the filter type and a helper method to check and return the type.